extensions/mssql/src/mssql.d.ts

```diff
@@ -885,6 +885,7 @@ export interface StopPerfDataCollectionResult {
 }
 
 export interface RefreshPerfDataCollectionResult {
+	isCollecting: boolean;
```

Comment:
@aasimkhan30, this one line is the only one that refers to any mssql code outside of sql-migration, since the rest we had already merged separately in PR #18315.

We're adding one more item to this return result. This requires no other changes, since this one extra item is already present (see [here](https://github.com/microsoft/sqltoolsservice/blob/8120858ad24652db405e23875ffe74515d6cd2c8/src/Microsoft.SqlTools.ServiceLayer/Migration/Contracts/PerfDataCollectionRequest.cs#L89)) in version `v3.0.0-release.202` of the sqltoolsservice, which was the version we upgraded to as part of PR #18315.